docs/en_US/Compressor/Overview.md

```diff
@@ -8,7 +8,6 @@ We have provided two naive compression algorithms and four popular ones for user
 |---|---|
```

Comment:
“We have provided two naive compression algorithms and four popular ones for users, including three pruning algorithms and three quantization algorithms:” this line should also be updated